### ISSUE TITLE
Tumblr OAuth 1.0a

### ISSUE THREAD

**Issue by romaonthego** [AFNetworking/AFOAuth1Client#15]
Hi Mattt, I can't make it work with Tumblr OAuth 1.0a (http://www.tumblr.com/docs/en/api/v2#auth), for some reason oauth_signature generated by AFOAuth1Client doesn't match value expected by Tumblr. Could you please advise, what can be possibly wrong?

Error Domain=AFNetworkingErrorDomain Code=-1011 "Expected status code in (200-299), got 401" UserInfo=0x9c6ad40 {NSLocalizedRecoverySuggestion=oauth_signature [9ExDk1efS/c3j3GNua/9FUu/jhI=] does not match expected value [8M3R+nuZ9FTBgqI1jb1eNJe7EQI=], AFNetworkingOperationFailingURLRequestErrorKey=<NSMutableURLRequest http://www.tumblr.com/oauth/access_token?oauth_consumer_key=ISY7GdYtCDXfzo4hSfWTlaAVmkRreLBttE15Igedfr4sfaAYxW&oauth_nonce=EEA112B3-B7F5-4119-B7BE-03A1B5FEDD2C&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1359338368&oauth_token=qDuc2OAcIyonkNP2tZlrj1IDWskGziDQRwlgx5nhlHgTYUygFo&oauth_verifier=XgpmG5hKTvIhXe1CEo9nn9TS8olfc0CWQXHk6fRebEeVN0WXyy&oauth_version=1.0&realm=http%3A%2F%2Fwww.tumblr.com%2F>, NSErrorFailingURLKey=http://www.tumblr.com/oauth/access_token?oauth_consumer_key=ISY7GdYtCDXfzo4hSfWTlaAVmkRreLBttE15Igedfr4sfaAYxW&oauth_nonce=EEA112B3-B7F5-4119-B7BE-03A1B5FEDD2C&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1359338368&oauth_token=qDuc2OAcIyonkNP2tZlrj1IDWskGziDQRwlgx5nhlHgTYUygFo&oauth_verifier=XgpmG5hKTvIhXe1CEo9nn9TS8olfc0CWQXHk6fRebEeVN0WXyy&oauth_version=1.0&realm=http%3A%2F%2Fwww.tumblr.com%2F, NSLocalizedDescription=Expected status code in (200-299), got 401, AFNetworkingOperationFailingURLResponseErrorKey=<NSHTTPURLResponse: 0xac4a2b0>}

My code:

``` objective-c
AFOAuth1Client *client = [[AFOAuth1Client alloc] initWithBaseURL:[NSURL URLWithString:@"http://www.tumblr.com"]
                                                             key:@"ISY7GdYtCDXfzo4hSfWTlaAVmkRreLBttE15Igedfr4sfaAYxW"
                                                          secret:@"JX994q7T9va9Rm6VNgrMw6hiLgllEQzhsB2T7TtXWhN5yYj2IZ"];

[client authorizeUsingOAuthWithRequestTokenPath:@"/oauth/request_token"
                          userAuthorizationPath:@"/oauth/authorize"
                                    callbackURL:[NSURL URLWithString:@"testapp://success"]
                                accessTokenPath:@"/oauth/access_token"
                                   accessMethod:@"GET"
                                        success:^(AFOAuth1Token *accessToken) {
                                            NSLog(@"Success: %@", accessToken);
                                        } failure:^(NSError *error) {
                                            NSLog(@"Error: %@", error);
                                        }];
```

**Comment by chrisallick** [comment]
I actually got this working, however my issue is that the query string returned to my app has s token and a verifier, however there is no method to create a new access token from that information so you cannot make the call to the class to create an access token...

**Comment by mattt** [comment]
This should be fixed with #22. Please let me know if you're still running into this after updating to the latest version of `AFOAuth1Client"
